demosplan/DemosPlanCoreBundle/Logic/Statement/DraftStatementFileHandler.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace demosplan\DemosPlanCoreBundle\Logic\Statement;
+
+use demosplan\DemosPlanCoreBundle\Logic\FileService;
+use demosplan\DemosPlanCoreBundle\Repository\DraftStatementFileRepository;
+
+class DraftStatementFileHandler
```

Comment:
once in a lifetime chance to provide tests for this class from the very beginning :-)

Reply:
Good idea!! I will make a PR for the tests